File: apps/web/fix_fleet.js
const fs = require('fs');
const file = 'src/app/dashboard/fleet/page.tsx';
let code = fs.readFileSync(file, 'utf8');

// There is an extra </div> at line 312
code = code.replace(/<s\.icon className=\{\`h-4 w-4 \$\{s\.color\}\`\} \/>\n            <\/div>\n            <p className=\{\`text-2xl font-bold tabular-nums \$\{s\.color\}\`\}>\{s\.value\}<\/p>\n            <\/div>\n          <\/div>/g, '<s.icon className={`h-4 w-4 ${s.color}`} />\n            </div>\n            <p className={`text-2xl font-bold tabular-nums ${s.color}`}>{s.value}</p>\n          </div>');

fs.writeFileSync(file, code);

File: apps/web/fix_fleet.py
import re

with open('src/app/dashboard/fleet/page.tsx', 'r') as f:
    text = f.read()

# Let's fix the end structure completely. Let's find from </table> downwards.
text = re.sub(r'</table>[\s\S]*?(?=\}\s*\)\}\n\s*</div>\n\s*\);\n\})', 
'''</table>
            </div> {/* closes overflow-auto */}

            {/* Table footer */}
            <div className="border-t border-border/40 bg-muted/10 px-6 py-4 text-xs font-semibold text-muted-foreground/70">
              Rāda {filtered.length} no {jobs.length} darb{jobs.length === 1 ? "u" : "iem"}
            </div>
          </div>
        ''', text)

with open('src/app/dashboard/fleet/page.tsx', 'w') as f:
    f.write(text)
